planex/cmd/fetch.py

```diff
@@ -194,6 +194,30 @@ def fetch_repo(url, resource):
     write_originfile(resource.path, repo.remotes.origin.url, sha)
 
 
+class UnsupportedScheme(Exception):
+    """UnsupportdScheme exception"""
```

Comment:
"UnsupportedScheme"

Reply:
Thanks, I have pushed the fix typo to depend-defactor